py-polars/src/expr/datetime.rs

```diff
@@ -158,7 +158,7 @@ impl PyExpr {
             )
             .into()
     }
-    fn duration_hours(&self) -> Self {
+    fn dt_total_hours(&self) -> Self {
```

Comment:
Looks like these are not on the Rust expression architecture yet - we can add that in a different PR!